docs/developer-docs/latest/setup-deployment-guides/configurations/optional/environment.md

```diff
@@ -115,7 +116,7 @@ With these configuration files the server will start on various ports depending
 
 ```bash
 yarn start                                   # uses host 127.0.0.1
-NODE_ENV=production yarn start               # uses host 0.0.0.0
+NODE_ENV=production yarn start               # uses host defined in .env. If not defined, uses 0.0.0.0
```

Comment:
@petersg83 the line above with `yarn start` refers to the root server.js file in the code block above, which only has the hard-coded host value and no `env` reference. Am I correct that this will only run on 127.0.0.1? Piwi and I discussed this afternoon that these examples are maybe still a little confusing? WDYT?

Reply:
You're right, I didn't see the example above!